Worktez/worktez
# feat(remainingUIMigration): Completed the remaining UI Migration

### Functionality:
Completed the remaining UI: create new team, update team and create new organization pages.

### Risk level:

### How to test:
ng serve
firebase emulators:start
Go to the my organization page check the create team and update team pop-up modals
